app/assets/stylesheets/findingaids.css.scss

```diff
@@ -109,6 +109,25 @@ h5.index_title > i {
   margin-right: .5em;
 }
 
+.result_ut{
+  font-weight: bold;
+}
+
+dt.blacklight-collection_ssm{
+  display:none;
+}
+dd.blacklight-collection_ssm{
+   padding-top: 10px;
+  padding-bottom: 5px;
+}
+.search_within{
+ 
+  font-weight: bold;
+}
+
+.request_item_note{
+  padding-top:10px;
+}
 @media (max-width: 480px) {
```

Comment:
@eshadatta Can you clean up the spacing here and leave a comment as to why and when you need to hide the label